web/tests/functional/review_status/test_review_status.py

```diff
@@ -543,14 +543,29 @@ def test_review_status_changes(self):
             "This is false positive.")
         self.assertIsNone(multi_unreviewed_report)
 
+        rule_filter = ReviewStatusRuleFilter(
+            reportHashes=[UNCOMMENTED_BUG_HASH])
+        review_status_rule_before = self._cc_client.getReviewStatusRules(
+            rule_filter, None, None, 0)[0]
+
         # Storing the report directory of the project inserts further reports
         # to the database with the same bug hashes. The review status of
         # reports without source code comment should get the default review
-        # status set earlier.
+        # status set earlier. In the meantime, the review status rule date must
+        # not change.
 
         test_project_name2 = 'review_status_change2'
         codechecker.store(codechecker_cfg, test_project_name2)
 
+        rule_filter = ReviewStatusRuleFilter(
```

Comment:
Rule filter could be reused from before?
Maybe give it a more distinct name like uncommented_rule_filter or something.

Reply:
As discussed ofline, leaving out the second might cause it to change meaning if a third on would be introduced.